src/test/java/fi/digitraffic/tis/vaco/rules/RuleResultsListenerTests.java

```diff
@@ -140,18 +151,45 @@ void deadLetterQueueTest() throws JsonProcessingException {
         String jsonString = """
                 {
                   "entry": { "publicId":"abc1"},
-                  "task": { "id":2, "publicId":"abc2"}
+                  "task": { "id":2, "publicId":"abc2", "name": "gtfs.canonical" }
                 }
             """;
 
         JsonNode message = objectMapper.readValue(jsonString, JsonNode.class);
-        Task task = ImmutableTask.of("dlqTest", 100).withPublicId("abc2");
+        Task task = ImmutableTask.of("gtfs.canonical", 100).withPublicId("abc2");
         given(taskService.markStatus(task, Status.FAILED)).willReturn(task);
         given(taskService.findTask(task.publicId())).willReturn(Optional.of(task));
-        Entry entry = ImmutableEntry.of("abc1", "dlq test entry", "gtfs", "http://example.fintraffic", Constants.FINTRAFFIC_BUSINESS_ID, false);
+
+        ImmutableEntry.Builder entryBuilder = TestObjects.anEntry("gtfs");
+        Entry entry = entryBuilder.addTasks(task)
+            .publicId("abc1")
+            .name("dlq test entry")
+            .format("gtfs")
+            .url("http://example.fintraffic")
+            .businessId(Constants.FINTRAFFIC_BUSINESS_ID)
+            .sendNotifications(false)
+            .build();
+
         given(entryService.findEntry(entry.publicId())).willReturn(Optional.of(entry));
         givenResultProcessingResultsInNewProcessingJobSubmission();
 
+        Finding finding = ImmutableFinding.builder()
+            .publicId(entry.publicId())
+            .source(task.name())
+            .taskId(2L)
+            .message("Entry ended up in dead letter queue ")
+            .severity(FindingSeverity.ERROR)
+            .build();
+
+        FindingRecord findingRecord = ImmutableFindingRecord.builder()
+            .id(500L)
+            .publicId(String.valueOf(UUID.randomUUID()))
```

Comment:
```suggestion
            .publicId(NanoIdUtils.randomNanoId()))
```

Remember to add the import as well :)